src/stories/containers/transparency-report/placeholders/transparency-empty-table.tsx

```diff
@@ -11,15 +11,18 @@ import { CustomButton } from '../../../components/custom-button/custom-button';
 interface Props {
   breakdown?: boolean;
   longCode: string;
+  isDelegate?: boolean;
 }
 
-export const TransparencyEmptyTable = ({ breakdown = false, longCode }: Props) => {
+export const TransparencyEmptyTable = ({ breakdown = false, longCode, isDelegate = false }: Props) => {
   const { isLight } = useThemeContext();
   const isTable = useMediaQuery(lightTheme.breakpoints.down('table_834'));
 
   const handleClickMakerburn = useCallback(() => {
-    window.open(`${MAKER_BURN_LINK}/${longCode}`, '_blank');
-  }, [longCode]);
+    isDelegate
+      ? window.open('https://makerburn.com/#/expenses/core-units/DELEGATES', '_blank')
+      : window.open(`${MAKER_BURN_LINK}/${longCode}`, '_blank');
+  }, [isDelegate, longCode]);
```

Comment:
This should be done using a link instead a click event so this can be handled as a link and all its benefits

Reply:
you are right , done